lib/extensions/ar_base.rb

```diff
@@ -13,12 +13,18 @@ def self.truncate
 
     def self.reindex
       _log.info("Reindexing table #{reindex_table_name}")
-      connection.reindex_table(reindex_table_name)
-      _log.info("Reindexing table #{reindex_table_name}")
+      result = connection.reindex_table(reindex_table_name)
+      _log.info("Completed Reindexing of table #{reindex_table_name} with result #{result.result_status}")
     end
 
     def self.reindex_table_name
       table_name
     end
+
+    def self.vacuum
+      _log.info("Vacuuming table #{table_name}")
+      result = connection.vacuum_full_table(table_name)
+      _log.info("Completed Vacuuming of table #{table_name} with result #{result.result_status}")
```

Comment:
What does `result` look like when it's logged here?

Reply:
@carbonin it's a 1 or a 0 hah